Dat257 75

Huffla opened the pull request:
Fixed issues with year-month-day being shown.
Rounded values so that they do not stick out of the graph.

Huffla committed:
Rounded values

Rounder values and made it only show the year isntead of the day aswell

Huffla committed:
Removed step setting

Huffla committed:
Added padding to grid

jakobschy commented on Client/Components/ComparisonComponent.razor:
```diff
@@ -1,20 +1,28 @@
             <RadzenSeriesDataLabels Visible="true" />
         </RadzenBarSeries>
-        <RadzenValueAxis Min="0" Max="@SetBarWidth()">
+            <RadzenValueAxis Min="0" Max="@SetBarWidth()" FormatString="{0:N}" >
```
Remove Indentation

jakobschy commented on Client/Components/ComparisonComponent.razor:
```diff
@@ -1,20 +1,28 @@
         </RadzenBarSeries>
-        <RadzenValueAxis Min="0" Max="@SetBarWidth()">
+            <RadzenValueAxis Min="0" Max="@SetBarWidth()" FormatString="{0:N}" >
+                
```
Remove New Lines

Huffla replied:
Done

jakobschy reviewed (dismissed):
Remove added Indentation and New Lines in ComparisonComponent.razor

Flaskhals68 commented:
Looks mostly good. 
Integer overflows are still unhandled and there are still some graphs where the labels are overflowing and covering parts of the courties names.

![image](https://github.com/filipkauffeldt/DAT257-group-8/assets/77638480/a9b49529-57d9-4d1b-b1c9-43152690dcd4)

Flaskhals68 commented:
Should the unit not be displayed where it says "Consumption"?

Huffla committed:
Update ComparisonComponent.razor

filipkauffeldt commented on Client/Components/ComparisonComponent.razor:
```diff
@@ -1,20 +1,25 @@
     <h4>@CountryComparison.Name uses @GetComparisonPercentage(ComparisonCountryValue, OriginCountryValue) @ResourceType than @CountryOrigin.Name </h4>
     <p>Unit: @Unit</p>
     <RadzenChart>
+           
```
Remove new line

filipkauffeldt commented on Client/Components/ComparisonComponent.razor:
```diff
@@ -1,20 +1,25 @@
 ﻿
 
 <div class="main">
+    <div>
```
Should probably indent content in `div` to keep consistent code style

Huffla committed:
Fixed bugs when showing statistics

Fixed:
If a value was 0 it caused integer overflow. Now shows value increas isntead of percentage.

Formatter:
Formatter that formats value to remove trailing zeroes and rounds up

Huffla committed:
Written tests for added code

Huffla committed:
Changed position of unit text

Added better visuals

Huffla committed:
Updated visuals on country of the day page

Huffla committed:
Formatted code a little bit

Huffla committed:
Merge branch 'main' into DAT257-75

Huffla committed:
Update ComparisonComponent.razor

Huffla commented:
Fixed issues with absurd negative numbers, and formatted the numbers in a way that looks better, rounded them to a few decimal places.
Switches position of the unit to where consumption is.
Changed colors of the comparison cards to better match the color scheme that we chose for the project.
Also changed so when a values is close to 0 the comparison is not in percent but instead in the amount of units.

Flaskhals68 commented on Client/Components/Compare.razor.css:
```diff
@@ -97,13 +97,18 @@
     text-align: center;
     border-radius: 40px;
-    box-shadow: 0 0 15px 1px #808080;
+    box-shadow: 0 0 15px 1px #057dcd;
```
This feel a bit distracting.
Maybe keep the original border and box-shadow.

Flaskhals68 commented on Client/Components/DashBoard.razor.css:
```diff
@@ -66,13 +66,18 @@
     text-align: center;
     border-radius: 40px;
-    box-shadow: 0 0 15px 1px #808080;
+    box-shadow: 0 0 15px 1px #057dcd;
```
This feel a bit distracting.
Maybe keep the original box-shadow and border?

Huffla committed:
Uppdated css

Flaskhals68 reviewed (approved):
Looks great!
